jdaviz/configs/mosviz/mosviz.ipynb

```diff
@@ -10,34 +10,20 @@
    "source": [
     "# PREFIX\n",
     "import pathlib\n",
-    "import tempfile\n",
-    "import uuid\n",
-    "import zipfile\n",
     "from jdaviz import Mosviz\n",
+    "from tempfile import mkdtemp\n",
+    "import zipfile\n",
     "\n",
     "data_path = pathlib.Path('DATA_FILENAME')\n",
     "\n",
     "if zipfile.is_zipfile(data_path):\n",
     "    with zipfile.ZipFile(data_path, 'r') as zipped_data:\n",
-    "        extract_path = pathlib.Path(tempfile.gettempdir()) / str(uuid.uuid4())\n",
+    "        extract_path = pathlib.Path(mkdtemp())\n",
     "        zipped_data.extractall(extract_path)\n",
     "        data_path = extract_path\n",
```

Comment:
I am going to argue that this whole block is undesirable. Let the users uncompress data themselves. This notebook should just take a path and open stuff. No fancy pre-processing.

Reply:
FYI: Reached out to @orifox who will consult with the other POs on whether we should keep or abandon this feature

Reply:
Looks like the POs agree, so I removed that block!